modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -0,0 +1,43 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-constructing-pipelines-using-pipeline-builder_{context}"]
+= Constructing Pipelines using the Pipeline Builder
+
+You can use the Pipeline Builder to:
```

Comment:
Maybe add a statement or prerequisite that this procedure requires access to the console.

Reply:
Modified this statement. Prerequisites with links have been added in the Assembly for this flow.